--- a/.conkyrc
+++ b/.conkyrc
@@ -63,7 +63,7 @@
 ${color grey}CPU Usage:$color $cpu% ${cpubar 6}
 ${color grey}Processes:$color $processes  ${color grey}Running:$color $running_processes
 $hr
-${color grey}Battery:$color ${battery BAT1}
+${color grey}Battery:$color ${battery BAT0}
 ${color grey}Wired Networking:$color ${addr eth0}
  ${color grey}Up:$color ${upspeed eth0} k/s${color grey} - Down:$color ${downspeed eth0} k/s
 ${color grey}Wireless Networking:$color ${addr wlan0}
